platform/pulpcore/app/models/task.py

```diff
@@ -163,6 +163,23 @@ class Task(Model):
     parent = models.ForeignKey("Task", null=True, related_name="spawned_tasks")
     worker = models.ForeignKey("Worker", null=True, related_name="tasks")
 
+    def set_worker(self, worker):
+        """
+        Set the worker Foreign Key on this Task, save it, and log output in warning cases.
+
+        This updates the :attr:`worker`.
```

Comment:
This could be the full path to the attribute. I don't think Sphinx will link this as is. If it had the full classpath it would link it I think. Maybe rendering the docs would be a good way to check the docstrings.